README.md

```diff
@@ -234,11 +258,10 @@ that for consistency across projects, you would use following tags
   for a given functionality implementation and/or method development
 - `dataset` - for datasets
 
-Ultimate goals
-==============
+## Ultimate goals
+
 
-Reduce demand for prima ballerina projects
-------------------------------------------
+### Reduce demand for prima ballerina projects
 
 **Problem**: Scientific software is often developed to gain citations for
 original publication through the use of the software implementing it.
```

Comment:
Also, I think section needs to expanded further on explaining the problem ( I get the gist of gist). We can add more details backing up this claim (citations or blog posts etc). If we can make this claim stronger, we convince the reader to take this up.